wire-ios-data-model/Source/Model/Conversation/ZMConversation+SystemMessages.swift

```diff
@@ -205,4 +205,27 @@ extension ZMConversation {
             timestamp: timestamp
         )
     }
+
+    public func appendMLSMigrationPotentialGapSystemMessage(sender: ZMUser, at timestamp: Date) {
+        guard let context = managedObjectContext else {
+            return
+        }
+
+        let previousLastMessage = lastMessage
+
+        self.appendSystemMessage(
+            type: .potentialGap,
```

Comment:
I think this type corresponds to `ZMSystemMessageTypePotentialGap` instead of the intended `ZMSystemMessageTypeMLSMigrationPotentialGap`

Reply:
Done!